File: LMS.Common/ViewModels/Permission/PermissionViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LMS.Common.ViewModels.Permission
{
    public class PermissionViewModel
    {
        public PermissionViewModel()
        {
            SchoolPermissions = new List<SchoolPermissionViewModel>();
            ClassPermissions = new List<ClassPermissionViewModel>();
            CoursePermissions = new List<CoursePermissionViewModel>();
        }
        public List<SchoolPermissionViewModel> SchoolPermissions { get; set; }
        public List<ClassPermissionViewModel> ClassPermissions { get; set; }
        public List<CoursePermissionViewModel> CoursePermissions { get; set; }
    }
}

File: LMS.App/ClientApp/src/root/interfaces/permission/permissionViewModel.ts
import { ClassPermissionViewModel } from "./classPermissionViewModel";
import { CoursePermissionViewModel } from "./coursePermissionViewModel";
import { SchoolPermissionViewModel } from "./schoolPermissionViewModel";

export interface PermissionViewModel
{
    schoolPermissions:SchoolPermissionViewModel[];
    classPermissions:ClassPermissionViewModel[];
    coursePermissions:CoursePermissionViewModel[];
}
